Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,18 @@
 import React from 'react';
 import './Button.css';
+
+const NumberButton = props => {
+    return (
+        <div className="numberContainer">
+            <div className="numberbutton">
+                <button className={`basicButtonStyle ${props.buttonStyle}`}>{props.info}</button>
+            </div>
+        </div>
```

Comment:
you've got quite a tangle of divs happening here, for every single button it looks like there are TWO container divs. there's definitely a more concise way to organize these button rows.